pkgs/by-name/gs/gsc/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchgit, cmake }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gsc";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "https://git.launchpad.net/gsc";
+    sha256 = "sha256-U5lP8tpk8Vfsg2GMC80Az/oHdmqHApYZFX01clT/moo=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  preConfigure = "
+    substituteInPlace gsc.c --replace '/usr/share/GSC' \\
+      '${placeholder "out"}/share/GSC'
+  ";
+
+  cmakeFlags = [ "-DCMAKE_INSTALL_PREFIX=${placeholder "out"}" ];
+
+  meta = with lib; {
+    homepage = "https://heasarc.gsfc.nasa.gov/w3browse/all/gsc.html";
+    description = "Hubble Guide Stars Catalog";
+    platforms = platforms.all;
+    license = licenses.cc-by-sa-40;
```

```suggestion
    license = unfree;
```
According to https://gsss.stsci.edu/Acknowledgements/DataCopyrights.htm
> The GSC-I is copyright (c) 1989-1996 by the Association of Universities for Research in Astronomy, Inc. All Rights Reserved. [[contact](mailto:mclean@stsci.edu?subject=GSC1%20copyright)]

I think that makes this `unfree`, sadly.